bibeasy/gsheet.py

```diff
@@ -63,13 +63,13 @@ def fetch_gsheet_contents(pubtypes):
                                                                 # a dict keyed by its contents
                                                                 # and greatly speeds up parsing.
     # union the multiple sheets into a single DataFrame
-    df_csv = pandas.DataFrame()
-    for pubtype, df_tmp in publications.items():
-        df_tmp['Type'] = pubtype
-        df_csv = df_csv.append((df_tmp), sort=False)
-        logging.info(f"  Total '{pubtype}' entries: {len(df_tmp)}")
-
-    return df_csv
+    pub_lst = []
+    for pubtype, item in publications.items():
+        item['Type'] = pubtype
+        pub_lst.append(item)
+        logging.info(f"  Total '{pubtype}' entries: {len(item)}")
```

Comment:
Tiny nit, but I think it might be good to keep the `df_` prefix, since otherwise it might not be clear that `item` is a DataFrame.

```suggestion
    for pubtype, df_pubs in publications.items():
        df_pubs['Type'] = pubtype
        pub_lst.append(df_pubs)
        logging.info(f"  Total '{pubtype}' entries: {len(df_pubs)}")
```